src/main/java/seedu/address/ui/HomeCard.java

```diff
@@ -0,0 +1,237 @@
+        contactAmount.setText(getContactAmount().toString());
+        XYChart.Series<String, Number> series = new XYChart.Series<>();
+        series.getData().add(new XYChart.Data<>(MONEY_OWED_LABEL, getTotalDebt())); // Replace 1500 with actual value
+        series.getData().add(new XYChart.Data<>(OWED_MONEY_LABEL, getTotalCredit())); // Replace 2000 with actual value
+        chart.getData().add(series);
+        chart.setCategoryGap(CATEGORY_GAP);
+        chart.setBarGap(BAR_GAP);
+        chart.setLegendVisible(false);
+        chartLabel.setText(CHART_LABEL);
+        availableToday.setText(AVAILABLE_LABEL);
+        setUpList();
+        setUpAnimation();
+    }
+
+    /**
+     * Sets up listView elements.
+     */
+    private void setUpList() {
+        availableList.setItems(getAvailableTodayList());
+        availableList.setCellFactory(listView -> new AvailableTodayCell());
+        availableList.setFocusTraversable(false);
+        availableList.setMouseTransparent(true);
+    }
+
+    private void setUpAnimation() {
+        moveTransition = new TranslateTransition(Duration.millis(MOVE_DURATION), this.getRoot());
+        double originalPosition = this.getRoot().getTranslateX();
+        moveTransition.setFromX(originalPosition + MOVE_INITIAL);
+        moveTransition.setToX(originalPosition - MOVE_POP);
+        moveBackTransition = new TranslateTransition(Duration.millis(MOVE_DURATION), this.getRoot());
+        moveBackTransition.setFromX(originalPosition - MOVE_POP);
+        moveBackTransition.setToX(originalPosition);
+        moveBackTransition.setDelay(Duration.millis(MOVE_DURATION));
+        fadeInTransition = new FadeTransition(Duration.millis(FADE_DURATION), this.getRoot());
+        fadeInTransition.setFromValue(FADE_INITIAL);
+        fadeInTransition.setToValue(FADE_ULTIMATE);
+        fadeInTransition.playFromStart();
+        moveTransition.playFromStart();
+        moveBackTransition.playFromStart();
+    }
+
+
+    /**
+     * @return List of people who are available today.
+     */
+    public ObservableList<Person> getAvailableTodayList() {
+
+        Day filterDay = Day.MONDAY;
+        DateTimeFormatter dayFormatter = DateTimeFormatter.ofPattern("EEEE");
+        String today = LocalDateTime.now().format(dayFormatter);
+        switch (today) {
+        case "Monday":
+            filterDay = Day.MONDAY;
+            break;
+        case "Tuesday":
+            filterDay = Day.TUESDAY;
+            break;
+        case "Wednesday":
+            filterDay = Day.WEDNESDAY;
+            break;
+        case "Thursday":
+            filterDay = Day.THURSDAY;
+            break;
+        case "Friday":
+            filterDay = Day.FRIDAY;
+            break;
+        case "Saturday":
+            filterDay = Day.SATURDAY;
+            break;
+        case "Sunday":
+            filterDay = Day.SUNDAY;
+            break;
+        default:
+            break;
+        }
```

Comment:
Do you want to use `Day::getDay` method? Then you can reduce this chunk as follows:

```suggestion
        DateTimeFormatter dayFormatter = DateTimeFormatter.ofPattern("EEEE");
        String today = LocalDateTime.now().format(dayFormatter);
        Day filterDay = Day.getDay(today);
```